doc/sphinx/source/pocketlab/pocketlab.rst

```diff
@@ -0,0 +1,55 @@
+============
+Introduction
+============
+Pocket lab is a portable display-based embedded solution created primarily for demoing the ADCs functionality.
+Pocket Lab makes use of `STM32F769NI-Discovery board <https://www.st.com/en/microcontrollers-microprocessors/stm32f769ni.html>`_ from ST microelectronics and 3rd party library LVGL.
+The demo functionality was primarily focused on device configuration, register memory access, data capture and FFT analysis.
+This documentation describes various components of pocket lab application in subsequent sections.
+
+.. Pocketlab Firmware Structure
+.. include:: /source/pocketlab/pocketlab_firmware_structure.rst
+
+=================
+Pocketlab Library
+=================
+
+Source Code: https://github.com/analogdevicesinc/precision-converters-library/tree/main/pocket_lab
```

Comment:
![image](https://github.com/analogdevicesinc/precision-converters-firmware/assets/118167755/17bd7db8-3690-41f3-a5f3-58f22ba59811)
There's a slight misalignment in the above links in the html page. 

```suggestion
Source Code: `Precision Converters Library <https://github.com/analogdevicesinc/precision-converters-library/tree/main/pocket_lab>`_
```

Can do same changes for below line as well.